plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSessionProperties.java

```diff
@@ -311,6 +313,16 @@ public IcebergSessionProperties(
                         "Enable sorted writing to tables with a specified sort order",
                         icebergConfig.isSortedWritingEnabled(),
                         false))
+                .add(integerProperty(
+                        MAX_SPLITS_PER_SECOND,
+                        "Throttle the maximum number of splits that can be assigned to tasks per second",
+                        icebergConfig.getMaxSplitsPerSecond(),
+                        false))
+                .add(integerProperty(
+                        MAX_OUTSTANDING_SPLITS,
+                        "Target number of buffered splits for each table scan in a query, before the scheduler tries to pause itself",
+                        icebergConfig.getMaxOutstandingSplits(),
+                        false))
```

Comment:
Not convinved we need this.
The PR quoted use-case (controlling HDFS namenode load) would ask for this not to be configurable by end user

Reply:
I've introduced this flag to be consistent with Hive, Delta and Hudi which all expose this config flag. Even though it's not the specific flag which addresses the PR-quoted use case, it was really easy to include this too in this PR, since it's needed for initializing the `AsyncQueue` anyway.